config/locales/client.en.yml

```diff
@@ -2234,21 +2234,10 @@ en:
         reaction: "new reaction"
         votes_released: "Vote was released"
 
-    upload_selector:
```

Comment:
I think this will force a re-translate of the subkeys. Not sure it's worth the rename.

Reply:
I think you are right, better to avoid re-translating, I've reverted renaming.